src/utils/Timer.ts

```diff
@@ -23,7 +23,7 @@ export class Timer extends Event<[void]> {
         const expiration = Timer.getEpochTime() + durationInSeconds;
         if (this.expiration === expiration && this._timerHandle) {
             // no need to reinitialize to same expiration, so bail out
-            logger.debug("skipping since already initialized for expiration at", this.expiration);
```

Comment:
This log is misleading since the check isn't necessarily enforcing that a duration value was already used once. e.g. this condition prevents a timer with duration with 10 seconds initialized 3 seconds ago from bing reinitialized when a new duration of 7 seconds is passed.

Now that I look at it, this whole early return seems unnecessary. I'm not sure what consequence it's avoiding.

Reply:
Yeah I think the old message reads fine. perhaps it could be formatted as a date string but it's not really important to me